opentelemetry-sdk/src/metrics/internal/exponential_histogram.rs

```diff
@@ -100,9 +100,20 @@ impl<T: Number> ExpoHistogramDataPoint<T> {
             if (self.scale - scale_delta as i8) < EXPO_MIN_SCALE {
                 // With a scale of -10 there is only two buckets for the whole range of f64 values.
                 // This can only happen if there is a max size of 1.
-                opentelemetry::global::handle_error(MetricsError::Other(
-                    "exponential histogram scale underflow".into(),
-                ));
+
+                // TODO - to check if this should be logged as an error if this is auto-recoverable.
+                otel_debug!(
+                    name: "ExponentialHistogramDataPoint.Scale.Underflow",
```

Comment:
It seems the naming convention is `CamelCase.` for log names. Is there a reason here?

Reply:
There are no specific reasons; it simply offers clearer visibility and effectively represents hierarchy when combined with dots. For example, `ExponentialHistogramDataPoint.Scale.Underflow` seems better to `exponentialhistogramdatapoint.scale.underflow` or `exponential_histogram_datapoint.scale.underflow`.
But happy to change if there are specific guidelines/preferences.

Reply:
@TommyCpp  - I am merging this to unblock. If there are specific recommendations, we can cover separately as this will affect throughout.

Reply:
We can get some inputs from OTel Events team recommendations. Our event name is same as what OTel Events are. (There are lot of ongoing discussions in this area, so we need to comeback to this once things are more settled)